### PR TITLE
Fix Oracle Functions and Remove "sanction" from Tron

### PR DESCRIPTION
**Pull request by TJ-Everett**
Rename restriction/sanction function names
Rename restriction/sanction errors

### CODE REVIEW

File: test/client/token/ERC20/integration/ApplicationERC20.t.sol [hunk 1]
```diff
@@ -233,12 +233,26 @@ contract ApplicationERC20Test is TestCommonFoundry {
...
+
+    /**
+     * @dev Test the oracle rule, both allow and deny types
+     */
+    function testOracleAddSingleAddressERC20() public {
+        /// Test adding single address to allow list 
+
+
+        /// Test adding single address to deny list 
+
```
Comment (oscarsernarosero) [inline test/client/token/ERC20/integration/ApplicationERC20.t.sol]: what is this for?

**Review by oscarsernarosero — COMMENTED** [review]
Looks good to me. I just have a question on a test. It looks like you were about to test something and then forgot about it.

File: src/common/ErrorSelectors.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ An online Keccak-256 hash digester can be found at https://emn178.github.io/onli
 | 0x5b2790b5 | AmountsAreZero()                                                 |
 | 0xd4c6aa81 | TokenInvalid(address)                                            |
 | 0xf1ec5eb7 | AmountExceedsBalance(uint256)                                    |
-| 0x6bdfffc0 | AddressIsRestricted()                                            |
+| 0x2767bda4 | AddressIsDenied()                                            |
```
Comment (ShaneDuncan602) [inline src/common/ErrorSelectors.md]: This is my OCD again. Can we fix the spacing on the end bar so it lines up?

File: test/client/token/ERC20/integration/ApplicationERC20.t.sol (hunk 1 above)
Comment (ShaneDuncan602) [reply test/client/token/ERC20/integration/ApplicationERC20.t.sol]: Yeah, it looks like this was missed.

Comment (TJ-Everett) [reply test/client/token/ERC20/integration/ApplicationERC20.t.sol]: That would be the place I was going to add additional tests and forgot too.

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
Looks good. Just need that additional test.

**Review by ShaneDuncan602 — APPROVED** [review]
Perfect. Great job.

**Review by oscarsernarosero — APPROVED** [review]
looks good!

**Pull request merged**